modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/segregated-comparison/segregated-linear-physics.i

```diff
@@ -0,0 +1,80 @@
+mu = 2.6
+rho = 1.0
+advected_interp_method = 'average'
+
+[Mesh]
+  [mesh]
+    type = CartesianMeshGenerator
+    dim = 2
+    dx = '0.3'
+    dy = '0.3'
+    ix = '3'
+    iy = '3'
+  []
+[]
+
+[Problem]
+  #TODO: remove both
+  linear_sys_names = 'u_system v_system pressure_system'
+  previous_nl_solution_required = true
+[]
+
+[Physics]
+  [NavierStokes]
+    [FlowLinear/flow]
```

Comment:
Could be FlowSegregated(Linear). We will remove the old segregated capabilities once everything has been migrated to the new one. Or something similar.

This is just a conversation starter, the name could probably stay.

Reply:
changed. FlowLinear was weird.